resources/views/settings/ldap.blade.php

```diff
@@ -91,6 +91,49 @@
                             </div>
                         </div>
 
+                        <!--  Default LDAP Permissions Group Select -->
+                        <div class="form-group{{ $errors->has('groups') ? ' has-error' : '' }}">
+                            <label class="col-md-3 control-label" for="groups[]"> {{ trans('general.groups') }}</label>
```

Comment:
We would only need `groups[]` here if we wanted to accept multiple options - which your HTML code does do, but if that's the case, we need to consider the migration above, since it looks like it wants a straight string.